Runtime/Core/ProBuilderMesh.cs

```diff
@@ -129,8 +136,8 @@ internal new MeshRenderer renderer
         {
             get
             {
-                if (m_MeshRenderer == null)
-                    m_MeshRenderer = GetComponent<MeshRenderer>();
+                if (!gameObject.TryGetComponent<MeshRenderer>(out m_MeshRenderer))
```

Comment:
Is there a reason why this called isn't inside a `if (m_MeshRenderer == null)` ?

Reply:
`m_MeshRenderer` can be deleted, this change was intended to make the `renderer` property no longer cached.